reinforcement_learning/agents/common/action_value_derived_policy.py

```diff
@@ -53,22 +50,4 @@ def _repr_svg_(self):
         return self.action_value._repr_svg_()
 
     def view(self):
-        return self.action_value.view()
-
-if __name__ == '__main__':
-    av = LazyTabularActionValue()
-
-    s = SimpleState([[-1, -1], [-1, 1]])
-
-    a1 = SimpleAction([0, 0])
-    a2 = SimpleAction([0, 1])
-    a3 = SimpleAction([1, 0])
-
-    av[s, a1] = 6
-    av[s, a2] = 2.9
-    av[s, a3] = -10
-
-    egp = ActionValueDerivedPolicy(av)
-
-    print(egp)
-    egp.view()
\ No newline at end of file
+        return self.action_value.view()
```

Comment:
It's also necessary, because only human-eay can test if graphical appearience is good enaugh...